C#/cs01/04.cs

```diff
@@ -9,9 +9,13 @@ namespace Brainchild.Curriculum.Solutions
     Fizz Buzz, 16, 17, Fizz, 19, Buzz, Fizz, 22, 23, Fizz, Buzz, 26, 
     Fizz, 28, 29, Fizz Buzz, 31, 32, Fizz, 34, Buzz, Fizz, ...
     */
-    public class FizzBuzzV2
+    public class FizzBuzzV2:FizzBuzzPrinter
```

Comment:
i mean compare the description  added  as comment
```
Write a program that prints the numbers from 1 to 100 and for multiples of '3' print "Fizz" instead of the number 
    and for the multiples of '5' print "Buzz". And if both Multiples of 3&5 ,print FizzBuzz	    and for the multiples of '5' print "Buzz". And if both Multiples of 3&5 ,print FizzBuzz
    1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, 	    1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, 
    Fizz Buzz, 16, 17, Fizz, 19, Buzz, Fizz, 22, 23, Fizz, Buzz, 26, 	    Fizz Buzz, 16, 17, Fizz, 19, Buzz, Fizz, 22, 23, Fizz, Buzz, 26, 
    Fizz, 28, 29, Fizz Buzz, 31, 32, Fizz, 34, Buzz, Fizz, ...	    Fizz, 28, 29, Fizz Buzz, 31, 32, Fizz, 34, Buzz, Fizz, ...
```